src/shared/sleep-config.c

```diff
@@ -102,6 +107,179 @@ int parse_sleep_config(SleepConfig **ret_sleep_config) {
         return 0;
 }
 
+/* If battery percentage capacity is less than equal to 5% return success */
+int battery_is_low(void) {
+        int r;
+
+        r = read_battery_capacity_percentage();
+        if (r == -ENOENT)
+               return false;
+        if (r < 0)
+               return r;
+
+        return r <= 5;
+        /* If battery current capacity percentage is equal to or less than 5%, return 1.
+         * We have not used battery capacity_level since value is set to full
+         * or Normal in case acpi is not working properly.
+         * In case of no battery 0 will be returned and system will be suspended for 1st cycle then hibernated */
```

Comment:
We usually locate such comment at the beginning of the function:
```c
int battery_is_low(void) {
        int r;

        /* comment */

        r = read_battery_capacity_percentage();
        ...
}
```

Reply:
ok. Fixed. Thanks!